src/graphql/root/query/btc-price-list.ts

```diff
@@ -0,0 +1,87 @@
+import moment from "moment"
+import { GT } from "@graphql/index"
+
+import { MS_PER_DAY } from "@config/app"
+
+import PricePoint from "@graphql/types/object/price-point"
+import PriceGraphRange, {
+  priceRangeValues,
+} from "@graphql/types/scalar/price-graph-range"
+
+import { getHourlyPrice } from "@services/local-cache"
+import { getCurrentPrice } from "@services/realtime-price"
+
+const rangeStartDateMS: (string: typeof priceRangeValues[number]) => number = (range) => {
+  const currentTimestamp = Date.now()
+
+  switch (range) {
+    case "ONE_DAY":
```

Comment:
this should have a default value

Reply:
The default value is intentionally left out here for static exhaustiveness